lib/activerecord-import/import.rb

```diff
@@ -941,7 +941,7 @@ def load_association_ids(model)
           association = association.target
           next if association.blank? || model.public_send(column_name).present?
 
-          association_primary_key = Array(association_reflection.association_primary_key)[column_index]
+          association_primary_key = Array(association_reflection.association_primary_key.tr("[]:", "").split(", "))[column_index]
```

Comment:
@fragkakis can you give a brief description of what is happening here and on line 999? I'm surprised in a good way that it seems like the necessary change is fairly small and focused 👍

Reply:
@jkowens if you think that this review is better held over at your repository, I can re-create the PR.